api/main.go

```diff
@@ -35,6 +35,16 @@ import (
 	"github.com/apisix/manager-api/log"
 )
 
+var Version string
+
+func printInfo() {
+	fmt.Fprint(os.Stdout, "The manager-api is running successfully!\n\n")
```

Comment:
Just keep version info short, welcome is unnecessary

Reply:
Sorry for the PR title. It's not only the version info. It also fixes #853.
And `-v` will be implemented at https://github.com/apache/apisix-dashboard/pull/773. Manager-api need a cli scafford.

Reply:
This is a different line with `listen on:`, this line dose not offer any helpful info to user.
The original issues demo is just only one line `manager-api is running, and listen on: http://127.0.0.1:8080.`
BTW: I think `manager-api is listen on : xxxx`  would be more simple,how do you think? @membphis @nic-chen